src/OrchardCore.Modules/OrchardCore.Users/Controllers/ResetPasswordController.cs

```diff
@@ -131,7 +131,7 @@ public async Task<IActionResult> ResetPassword(ResetPasswordViewModel model)
 
             if (TryValidateModel(model) && ModelState.IsValid)
             {
-                if (await _userService.ResetPasswordAsync(model.Email, Encoding.UTF8.GetString(Convert.FromBase64String(model.ResetToken)), model.NewPassword, (key, message) => ModelState.AddModelError(key, message)))
+                if (await _userService.ResetPasswordAsync(model.Email, Encoding.UTF8.GetString(Convert.FromBase64String(model.ResetToken)), model.NewPassword, (key, message) => ModelState.AddModelError(key == "Password" ? nameof(ResetPasswordViewModel.NewPassword) : key, message)))
```

Comment:
Can we use `nameof` instead of magic string

Reply:
I was using "Password" based on what UserService does [here](https://github.com/OrchardCMS/OrchardCore/blob/c476a3df796cb79d095fe767edbd4e10d4a27c75/src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs#L243). That's really the main problem here is UserService assumes your field is named "Password".